integreat_cms/cms/templates/pages/page_tree.html

```diff
@@ -160,23 +160,23 @@ <h1 class="heading">
                     </option>
                     {% if request.user.expert_mode %}
                         {% with language_translated_name=language.translated_name %}
-                            {% blocktrans asvar xliff_export_all %}Export selected pages for {{ language_translated_name }} translation{% endblocktrans %}
-                            {% blocktrans asvar xliff_export_public %}Export only published pages for {{ language_translated_name }} translation{% endblocktrans %}
+                            {% blocktrans asvar xliff_export_public %}Export published pages for {{ language_translated_name }} translation{% endblocktrans %}
+                            {% blocktrans asvar xliff_export_all %}Export unpublished⚠️ and published pages for {{ language_translated_name }} translation{% endblocktrans %}
```

Comment:
```suggestion
                            {% blocktrans asvar xliff_export_all %}Export unpublished (⚠️) and published pages for {{ language_translated_name }} translation{% endblocktrans %}
```
I personally find the placement of the ⚠️ a bit weird, but not sure if my suggestion is an improvement^^

Reply:
I agree that this looks a little better :)